configs/peer/genesis.json

```diff
@@ -53,8 +54,8 @@
                       }
                     },
                     "value_type": "Quantity",
-                    "metadata": {},
-                    "mintable": true
+                    "mintable": "Infinitely",
```

Comment:
Will cause conflicts with #2104

Reply:
will it conflict if it's exactly the same?

Reply:
Or if it's not changed.